packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/loadingaconfiguration.md

```diff
@@ -0,0 +1,48 @@
+
+Create a configuration by calling `IdxClientConfiguration`. You can set the values directly in your code or load them from a file. This code shows loading the values from a property file in your project.
```

Comment:
"You can set the values directly in your code or load them from a file" 
This repeats a sentence in the para immediately above here in index.md. You'll need to delete this or the one above.

Reply:
fixed in current version